src/main/java/com/anhtester/driver/DriverManager.java

```
@@ -19,9 +19,7 @@ public static void setDriver(WebDriver driver) {
     }
 
     public static void quit() {
```

As per Google coding standards, this is incorrect. Empty method without implementation. Methods should have proper implementation or throw UnsupportedOperationException.